tools/unitctl/README.md

```diff
@@ -157,6 +157,20 @@ Imported /opt/unit/config/put.json -> /config
 Imported 3 files
 ```
 
+### Export configuration from a running Unit instance
+```
+$ unitctl export -f config.tar
+```
+
+Addtionally, standard out can be used:
+```
+$ unitctl export -f -
+$ unitctl export -f - | tar xf - config.json
+$ unitctl export -f - > config.tar
+```
+
+*Note:* The exported configuration will be devoid of any installed certificates. The Unit configuration API does not support fetching certificates or private keys once installed. User's will need to add relevant certificates and keys to an export.
```

Comment:
```suggestion
*Note:* The exported configuration omits certificates. 
```

...and then can we document importing the configuration (including the optional step of manually adding certs?)

It feels like there's a _lot_ more unitctl could do to ease cert management here, but that's best left for a future PR.

Reply:
Sure. I can reword this accordingly. Certificates are imported with `unitctl import` in an example in the readme, but I agree that all the docs need to be elaborated on.